cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -589,6 +588,13 @@ def _validate(self, mount_dir_list):
                 ),
                 FailureLevel.ERROR,
             )
+        for local_mount_dir in local_mount_dir_list:
+            if local_mount_dir in shared_mount_dir_list:
+                self._add_failure(
+                    f"Ephemeral drive mount directory `{local_mount_dir}` is overwritten by shared storage "
+                    f"mount directory. Please consider changing either directory to avoid the overwrite.",
+                    FailureLevel.WARNING,
```

Comment:
**Product decision alert**
I think that this kind of failure should be an error rather than a warning. Overwriting a local mount point should block the cluster creation in the same way a shared mount point overwrites another shared mount point.

Pending product decision here.

Reply:
Product decision is to be more conservative and keep the warning level.